# Sequence of Actions to be executed by robot
hri_client/Action[] action_sequence
